### PR TITLE
Get modernizr build working

### PR DESCRIPTION
**Pull request by dpoirier**
We want to use modernizr in a project, and this
gulpfile appears to support building a custom
modernizr based on what use we are making of it.
So we tried it.

Unfortunately, this required a major refactoring
of gulpful.js, to make sure we had built modernizr.js
before we tried to compile our javascript.  On
the plus side, it's more gulpish now.

Branch: modernizr

### CODE REVIEW

File: gulpfile.js [hunk 1]
```diff
@@ -111,37 +134,47 @@ var browserifyTask = function (options) {
...
+  return rebundle();
 };
+gulp.task('browserify', ['modernizr'], browserifyTask);
 
-var cssTask = function (options) {
+var cssTask = function () {
     var lessOpts = {
       relativeUrls: true,
     };
+    var start = new Date();
```
Comment (nmashton) [inline gulpfile.js]: Why did you move this up here?

File: gulpfile.js [hunk 2]
```diff
@@ -160,14 +193,9 @@ function rebuild(options) {
...
 }
 
-// Starts our development workflow
-gulp.task('default', function (cb) {
-  livereload.listen();
-
-  rebuild({
-    development: true,
-  });
+gulp.task('rebuild', ['css', 'browserify'])
```
Comment (nmashton) [inline gulpfile.js]: Duplicate of line 177?

File: gulpfile.js (hunk 1 above)
Comment (dpoirier) [inline gulpfile.js]: It looks like I added a log message to production mode saying how long it took.  But I don't see any reason to include that change here, I'll back it out.

**Comment by nmashton** [comment]
Aside from my two comments: 🚀

File: gulpfile.js (hunk 2 above)
Comment (dpoirier) [inline gulpfile.js]: Yes, and we don't appear to need the rebuild function anymore either.
